docs/installation.md

```diff
@@ -52,7 +52,7 @@ Edit your `pom.xml` file to add settings to the maven-compiler-plugin:
           <target>8</target>
           <compilerArgs>
             <arg>-XDcompilePolicy=simple</arg>
-            <arg>-Xplugin:ErrorProne</arg>
+            <arg>-Xplugin:ErrorProne -Xep:DeadException:ERROR</arg>
```

Comment:
This section is intended to provide a recommended default way to configure installs, this could be read as a recommendation to literally pass `-Xep:DeadException:ERROR` and not an example of how to place `-Xplugin` flags.